config/initializers/statsd.rb

```diff
@@ -5,6 +5,8 @@ class << Samson
   attr_accessor :statsd
 end
 
+NOT_APPLICABLE = 0
```

Comment:
should use that inside of DeployMetrics so they are linked together via this constant